### DISCUSSION TITLE
Any way to move the close button on right?

### DISCUSSION THREAD

**Discussion by Esarve** [piroor/treestyletab#3275] [Q&A] [answered]
Is there any way to move the close button to right? Currently I've added a css snippet which lets me enable the close button on hover. But I want to move it to the right. How to do that?

**Comment by piroor** [comment, marked as answer]
There is a code snippet: [Put closebox right side, even if I choose "Right side" style](https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#put-closebox-right-side-even-if-i-choose-right-side-style-1387). Did you try it?

**Reply by Esarve** [reply]
Sorry for the late reply. Yes it worked. I missed it somehow.
